backend/statistics/bq-schema/episodes.json

```diff
@@ -55,6 +55,10 @@
     "name": "TagIDs",
     "type": "JSON"
   },
+  {
```

Comment:
this file is maybe not used?

Reply:
No, it's not really used. It's not as easy to use as I initally expected and didn't see the value in spending the extra time.